Sources/SwiftParser/Parser.swift

```diff
@@ -585,3 +585,24 @@ extension SyntaxText {
     try body(UnsafeBufferPointer<UInt8>(start: self.baseAddress, count: self.count))
   }
 }
+
+extension Parser {
+  /// Consume a period as part of a member expression/type. Create unexpected +
+  /// missing tokens if there's trailing whitespace no leading whitespace
+  /// (whether from `leading` or from the leading whitspace of the current
+  /// token).
+  mutating func consumeMemberPeriod<R: RawSyntaxNodeProtocol>(leading: R?) -> (unexpected: RawUnexpectedNodesSyntax?, period: RawTokenSyntax) {
```

Comment:
Would this better be named `previousNode`. Leading sounds too much like the variable contains trivia to me.

Reply:
Sure, I can do that.